README.md

```diff
@@ -1,2 +1,15 @@
 # snippets
 This is a repo dedicated to snippets of code created to help with miscellaneous things.
+
+## chiliproject_migrate.rb
+This script is used for exporting everything related to a chiliproject project.
+To use this you first must from the project id of the project you are trying to migrate.
+The project id in the script must be replaced with the project id of the project you intend to export.
+This script creates the files in a directory specified by `$dump_dir` at the top of the script.
+Be sure to create this directory before running the script.
+Then, run `./script/console production` in the chiliproject root directory to gain access to the ORM console.
```

Comment:
Actually, it's the standard Ruby REPL with all production settings loaded (and hence pointed to the production database)